### ISSUE TITLE
Problems with login 

### ISSUE THREAD

**Issue by StellaASchlotter** [opencv/cvat#6391]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Steps to Reproduce (for bugs)
I used the latest develop branch and tried deploying this locally inside WSL2. I can access the site at localhost:8080. However even after I successfully created a superuser I still can't login. I always get the error:
```
Error: Request failed with status code 400. {"non_field_errors":["Unable to log in with provided credentials."]}.
@ cvat-app.tsx:357
```

### Expected Behaviour
I expected that to work as it did with previous versions.

### Your Environment

- Git hash commit (`git log -1`): commit f6420eb709cabf7a828c4b7c6dcc8a123fea2126
- Docker version `docker version` (e.g. Docker 17.0.05): 24.0.2
- Operating System and version (e.g. Linux, Windows, MacOS): Windows but WSL2 with Ubuntu 22:04

**Comment by StellaASchlotter** [comment]
I found this maybe relevant StackOverflow article about it: https://stackoverflow.com/questions/40076254/drf-auth-token-non-field-errors-unable-to-log-in-with-provided-credential

**Comment by StellaASchlotter** [comment]
I did the same thing on another PC and I dont get the problem.  Docker, compose and WSL is exactly the same version. I did a `docker system prune` before pulling and starting the images again. I am really lost on why this problem exist. 

Could this be a firewall issue even though I can access the login page?

**Comment by bsekachev** [comment]
I work with the latest develop branch of CVAT & the latest release on Ubuntu 22 under WSL2.
There was not any problems like you described. So, some more information is necessary. 

Logs, commands you did to install CVAT, response/request body, server configuration, any other.

**Comment by grzleadams** [comment]
We're seeing this as well, but on a K8s deployment via Helm (on Linux nodes). 

We have two identical (which I'm sure about, because I wrote a Terraform module to deploy CVAT instances, where the only difference is name/hostname) deployments, where one (which was deployed on an older version of CVAT and upgraded) allows auth and the other (deployed fresh on 2.4.2) doesn't. The only appreciable difference between the two (other than the upgrade path vs. clean deployment) is hostname (the one that works is `cvat.<domain>` and the other is `cvat-test.<domain>`. 

I'll try a clean deployment with develop/latest to see if it works.

**Comment by grzleadams** [comment]
> I found this maybe relevant StackOverflow article about it: https://stackoverflow.com/questions/40076254/drf-auth-token-non-field-errors-unable-to-log-in-with-provided-credential

This is interesting to me given the scenario I described above... existing users should (theoretically) not be impacted if they've already been serialized and written to the DB, whereas new users/fresh deployments would be impacted. Am I understanding that correctly?

**Comment by grzleadams** [comment]
develop/latest didn't resolve the issue. Also, I should note that we use LDAP authentication, and that's what's failing. If I create a local user on the Admin page, I can log in as that user.

**Comment by bsekachev** [comment]
@StellaASchlotter Hope you solved the issue. 
@grzleadams Unfortunately, we can't suggest how to solve issues with any custom modifications, like LDAP.

**Comment by bsekachev** [comment]
@StellaASchlotter Hope you solved the issue. 
@grzleadams Unfortunately, we can't suggest how to solve issues with any custom modifications, like LDAP.

**Comment by grzleadams** [comment]
@bsekachev That's fair, but if this is a serializer issue (as suggested in the SO question linked above) wouldn't it affect _all_ external auth, not just LDAP?

**Comment by grzleadams** [comment]
Following up on my question from yesterday, I threw debug logging into settings.py for both django_auth_ldap (a new logger) _and_ django (modified the existing logger level), and it doesn't look like the request is even getting past the `/api/auth/login` call (i.e., it's not django_auth_ldap causing a problem); there is nothing LDAP-related in the logs. 

I was looking at the [API schema](https://github.com/opencv/cvat/blob/1ef561a31a1fa0569440f5b56bb978b6d746fbbe/cvat/schema.yml#L108) and noticed the request body is passed from the serializer, which further leads me to believe something is up with that component. Really sorry to be bugging you like this but I don't want to have all my domain users locked out on the next upgrade/redeploy. Can we reopen this?

**Comment by grzleadams** [comment]
Nevermind... I used `diffsettings` to dump the Django settings for both deployments and for some reason the redeployment doesn't have the settings.py overrides in it at all. DJANGO_SETTINGS_MODULE probably just needs to be set.

**Comment by sunyd1006** [comment]
does this issue be resolved? i found the same problem at 0.11.0 version
